### PR TITLE
Updated RDSDeletionProtection.py for aws_db_instance

### PR DESCRIPTION
**Pull request by Jhanvi12**
I have updated rule CKV_AWS_139 (File -> RDSDeletionProtection.py) to check deletion protection for aws_db_instance also.

### CODE REVIEW

**Review by JamesWoolfenden — CHANGES_REQUESTED** [review]
Needs associate tests and new testcase/data for each

File: checkov/terraform/checks/resource/aws/IAMStarResourcePolicyDocument.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.common.util.type_forcers import force_list, extract_policy_dict
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from typing import List
+
+
+class IAMStarResourcePolicyDocument(BaseResourceCheck):
```
Comment (gruebel) [inline checkov/terraform/checks/resource/aws/IAMStarResourcePolicyDocument.py:7]: this looks like a wrong addition 😄

File: checkov/terraform/checks/resource/aws/RDSDeletionProtection.py [hunk 2]
```diff
@@ -7,7 +7,7 @@ class RDSDeletionProtection(BaseResourceValueCheck):
     def __init__(self):
         name = "Ensure that RDS clusters have deletion protection enabled"
         id = "CKV_AWS_139"
-        supported_resources = ['aws_rds_cluster']
+        supported_resources = ['aws_rds_cluster' ,'aws_db_instance']
```
Comment (gruebel) [inline checkov/terraform/checks/resource/aws/RDSDeletionProtection.py:10]: @JamesWoolfenden what do you think? looking at the name of the check we would either need to change/extend the name or create a separate check for it.

Comment (JamesWoolfenden) [reply checkov/terraform/checks/resource/aws/RDSDeletionProtection.py:10]: id go with a separate check one for db instance, we went that way for encryption and CMK checks and it lets have the control on suppression.

Comment (JamesWoolfenden) [reply checkov/terraform/checks/resource/aws/RDSDeletionProtection.py:10]: a new check for 'aws_db_instance' and new test cases and test for that new check

**Comment by Jhanvi12** [comment]
The rule is AWS DB Instance deletion protection .In need to merge from my local branch master to main bridgecrew master branch

**Comment by Jhanvi12** [comment]
> Needs associate tests and new testcase/data for each

Hello created test case for that , the link is https://github.com/Jhanvi12/checkov/blob/master/tests/terraform/checks/resource/aws/test_AWSDatabaseInstanceDeletionProtection.py

**Comment by JamesWoolfenden** [comment]
you're trying to merge into the wrong branch, thats why theres so many commit, main is your target branch.

**Pull request closed without merging**